ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  to: ironic-image
+promotion:
+  name: "4.2"
+  namespace: ocp
+resources:
```

Comment:
So I think we're going to update the repos: see https://github.com/openshift/release/pull/4411/files raw_steps definition.